### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ViktarBaloshka**

### CODE REVIEW

File: src/main/java/com/helloworld/helloworld/controller/messageController.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("message")
+public class messageController {
+    private int counter = 4;
+    private List<Map<String, String>> messages = new ArrayList<>() {{
```
Comment (YanushkevichVasili) [inline src/main/java/com/helloworld/helloworld/controller/messageController.java:15]: use the api 
List.of(...)

File: src/main/java/com/helloworld/helloworld/controller/messageController.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        add(new HashMap<String, String>() {{
```
Comment (YanushkevichVasili) [inline src/main/java/com/helloworld/helloworld/controller/messageController.java:16]: use other api Map.of(..,..)

File: src/main/java/com/helloworld/helloworld/controller/messageController.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("message")
+public class messageController {
```
Comment (YanushkevichVasili) [inline src/main/java/com/helloworld/helloworld/controller/messageController.java:13]: create  a SERVICE and move all logic to this service. controller should call this service and return data

File: src/main/java/com/helloworld/helloworld/controller/messageController.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        return messages;
+    }
+
+    @GetMapping("{id}")
+    public Map<String, String> getOne(@PathVariable String id) {
+        return getMessage(id);
+    }
+
+    private Map<String, String> getMessage(String id) {
+        return messages.stream().filter(messages -> messages.get("id").equals(id)).findFirst().orElseThrow(NotFoundException::new);
```
Comment (YanushkevichVasili) [inline src/main/java/com/helloworld/helloworld/controller/messageController.java:41]: The format rule is that each command in stream should be in new line

Comment (ViktarBaloshka) [reply src/main/java/com/helloworld/helloworld/controller/messageController.java:41]: It's done

**Pull request merged**